LICENSE.md

```diff
@@ -1,4 +1,4 @@
-Copyright (c) 2014 The KBase Project and its Contributors
+Copyright (c) 2014-present The KBase Project and its Contributors
```

Comment:
Ugh, I hate to ask, but is there an official doc on this? Do we update to the current year, or is it enough to put `-present`?

Reply:
With a quick look I can't find anything either way. I doubt it particularly matters given the looseness of the license. If you're really exercised about it we should ask legal

Reply:
I don't really care that much, TBH. I'll let someone else bother legal. :)